conda_docker/conda.py

```diff
@@ -417,7 +426,7 @@ def chroot_install(
     targ_record_fns = []
     for record in records:
```

Comment:
Maybe add a `can_link()` function and then use this once before the loop starts.

```py
def _can_link(platform, source_dir, target_dir):
    """Determines if we can link from source to target directory"""
    if platform.startswith("win"):
        return False
    src = os.path.join(source_dir, "__nightly_test__")
    trg = os.path.join(target_dir, "__nightky_test__")
    try:
        with open(src, "w"):
            pass
        os.link(src, trg)
        linkable = True
    except OSError:
        linkable = False
    finally:
        if os.path.isfile(trg):
            os.remove(trg)
        if os.path.isfile(src):
            os.remove(src)
    return linkable
```

Reply:
Done!